### ISSUE TITLE
mxnet fail to import onnx model

### ISSUE THREAD

**Issue by photoszzt** [apache/incubator-mxnet#18923]
## Description
mxnet fail to import onnx model exported by pytorch. The model is from https://github.com/biubug6/Pytorch_Retinaface

### Error Message
(Paste the complete error message. Please also include stack trace by setting environment variable `DMLC_LOG_STACK_TRACE_DEPTH=10` before running your script.)

  File "/disk/zhitingz/video-pipeline/sprocket/platform/lambda_program/onnxruntime/detection/detect.py", line 139, in mx_init
    sym, arg_params, aux_params = onnx_mxnet.import_model(mpath)
  File "/disk/zhitingz/mxnet-cpu/lib/python3.6/site-packages/mxnet/contrib/onnx/onnx2mx/import_model.py", line 59, in import_model
    sym, arg_params, aux_params = graph.from_onnx(model_proto.graph)
  File "/disk/zhitingz/mxnet-cpu/lib/python3.6/site-packages/mxnet/contrib/onnx/onnx2mx/import_onnx.py", line 115, in from_onnx
    inputs = [self._nodes[i] for i in node.input]
  File "/disk/zhitingz/mxnet-cpu/lib/python3.6/site-packages/mxnet/contrib/onnx/onnx2mx/import_onnx.py", line 115, in <listcomp>
    inputs = [self._nodes[i] for i in node.input]
KeyError: '1'

## To Reproduce
(If you developed your own code, please provide a short script that reproduces the error. For existing examples, please provide link.)
Get the exported onnx model from the above link. 
```
from mxnet.contrib import onnx as onnx_mxnet
sym, arg_params, aux_params = onnx_mxnet.import_model(mpath)
```

## What have you tried to solve it?

1. I try to print the node.input. It's ['input0', '1'].

**Comment by photoszzt** [comment]
I update the issue with my forked repo that contains some helpful script to reproduce this issue.

**Comment by szha** [comment]
cc @josephevans and @ChaiBapchya

**Comment by TristonC** [comment]
@Zha0q1 Have MXNet 1.x made any progress for this issue?

**Comment by Zha0q1** [comment]
no.. the recent onnx development has been focused on exporting rather than importing

**Comment by TristonC** [comment]
Thanks, @Zha0q1. Import might be more important if a user  want to test the performance on different frameworks.   @szha @sandeep-krishnamurthy Do we have any project to imporve the ONNX model importing?

**Comment by Joe2loft** [comment]
@photoszzt @Zha0q1   you can add args  keep_initializers_as_inputs=True  when you  run   torch.onnx.export  function.  
 but  after that,  i meet another problem -----  mxnet dont have resize op .
i found it's because there's a  torch.nn.functional.interpolate  function in retinaface,  which will be transform to resize op   by onnx, but mxnet dont have resize op.
i tried to add a  resize op , but i'm not familiar with  mxnet.
